Library/Homebrew/cmd/commands.rb

```diff
@@ -8,6 +8,14 @@ module Homebrew
   module_function
 
   def commands
+    require "options_for_brew_commands"
+    options do
+      command __method__.to_s
```

Comment:
`options(:commands) do` has a bit of duplication but reads nicer. @reitermarkus @alyssais, any thoughts on clever metaprogramming we could do here?